CONTRIBUTING.rst

```diff
@@ -5,28 +5,29 @@ Contributing
 Contributions are welcome and appreciated!
 Every little bit helps, and credit will always be given.
 
-If you are new to ScanCode and want to find easy tickets to work on, check https://github.com/nexB/scancode-toolkit/labels/easy
+.. _issue : https://github.com/nexB/scancode-toolkit/issue
+__ issue_
+If you are new to ScanCode and want to find easy tickets to work on, check `easy issues <https://github.com/nexB/scancode-toolkit/labels/easy>`_
 
 When contributing to ScanCode (such as code, bugs, documentation, etc.) you
-agree to the Developer Certificate of Origin http://developercertificate.org/
-and the ScanCode license (see the NOTICE file).  The same approach is used
+agree to the Developer `Certificate of Origin <http://developercertificate.org/>`_
+and the ScanCode license (see the `NOTICE <https://github.com/nexB/scancode-toolkit/blob/develop/NOTICE>`_ file).  The same approach is used
 by the Linux Kernel developers and several other projects.
 
 For commits, it is best to simply add a line like this to your commit message,
 with your name and email::
 
     Signed-off-by: Jane Doe <developer@example.com>
 
-Please try to write a good commit message.
-See https://github.com/nexB/aboutcode/wiki/Writing-good-commit-messages for details.
+Please try to write a good commit message, see `good commit message wiki <https://github.com/nexB/aboutcode/wiki/Writing-good-commit-messages>`_ for details.
+For writing good Pull Request messages and comments with markdown, read `this GitHub guide <https://guides.github.com/features/mastering-markdown/>`_ for once
```

Comment:
One tiny thing, I am not sure we want to use markdown in commit messages ...

Reply:
Here my reference is with pull request message, sorry for wrong understanding.